### PR TITLE
Fix wrong links on the stataic pages

### LINKED ISSUE #3659: Inconsistent behaviour of multistore on next.storefrontloud.io
## Steps to reproduce the issue
1. Open https://next.storefrontcloud.io/it
2. Open https://next.storefrontcloud.io/de
3. Click on language selector in the footer

## Current behavior
/de shop opens from the link, but with content patrially in german, partially in english. 
/it shop returns "something went wrong" error. 
language selector opened from /de store contains only 1 option
nextDe

## Expected behavior
If multistore is correctly enabled, then:
/de shop should contain all existing translations AND /it shop should correctly open, AND  language selector should contain all available options

if multistore is disabled, then /de shop should not be accessible

## Additional info
Problem seems to be either in config on next, or deployed version. During local test on release branch, behaviour of multistore is correct and consistent. 

## Repository
next.storefrontcloud.io

## Can you handle fixing this bug by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [x] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details
- Browser: latest Chrome
- OS: Ubuntu, Win10
- Code Version: release/1.11

### PR DESCRIPTION
**Pull request by andrzejewsky**
### Related issues
closes #3659

### Which environment this relates to
- [x] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [x] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance
- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: core/lib/multistore.ts [hunk 1]
```diff
@@ -217,3 +217,11 @@ export function localizedRoutePath (path: string, storeCode: string): string {
 
   return `/${storeCode}/${_path}`
 }
+
+export function localizedRouteOnlyForDefault (path: string) {
```
Comment (pkarw) [inline core/lib/multistore.ts]: I think this is a workaround only for this footr thing. I'd suggest solving this on theme level - this is misleading helper. It should be defined only in the `Category.vue`

**Comment by andrzejewsky** [comment]
@pkarw @patzick Completely agree, but please look at files, I used `localizedRouteOnlyForDefault` in two files (`Footer.vue`, `Static.vue`), that's why I've moved this logic to the function as I didn't want to copy-paste same logic.

**Comment by pkarw** [comment]
@andrzejewsky please just create a separate helper for that - not in multistore.ts

**Comment by filrak** [comment]
Everything is fine except things mentioned by @pkarw

**Pull request merged**